core/storage/collection/gae_models.py

```diff
@@ -47,6 +47,102 @@ def get_deletion_policy():
         return base_models.DELETION_POLICY.NOT_APPLICABLE
 
 
+class CollectionCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
+    """Log of commits to collections.
+
+    A new instance of this model is created and saved every time a commit to
+    CollectionModel or CollectionRightsModel occurs.
+
+    The id for this model is of the form 'collection-[collection_id]-[version]'.
+    """
+
+    # The id of the collection being edited.
+    collection_id = (
+        datastore_services.StringProperty(indexed=True, required=True))
+
+    @staticmethod
+    def get_deletion_policy():
+        """Model contains data to pseudonymize or delete corresponding
+        to a user: user_id field.
+        """
+        return (
+            base_models.DELETION_POLICY.PSEUDONYMIZE_IF_PUBLIC_DELETE_IF_PRIVATE
+        )
+
+    @staticmethod
+    def get_model_association_to_user():
+        """The history of commits is not relevant for the purposes of
+        Takeout, since commits do not contain any personal user data.
```

Comment:
```suggestion
        """The history of commits is not relevant for the purposes of
        Takeout since commits don't contain relevant data
        corresponding to users.
```

Reply:
Done.